manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Wacom.WacomTabletDriver
+PackageVersion: 6.3.45-1
+InstallerSwitches:
+  Silent: /s
+  SilentWithProgress: /s
+InstallerSuccessCodes:
+- 1
+- 2
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```
It's failing on x86 VM; so it's better to use `x64` here:
![image](https://user-images.githubusercontent.com/11600822/149168452-4e5f7970-ebf4-4767-83cb-2492fae1f764.png)
![image](https://user-images.githubusercontent.com/11600822/149168517-9ec9f3ca-ec7d-468c-a94a-4981795d51df.png)